documentation/API-linting-Implementation-Guideline.md

```diff
@@ -0,0 +1,76 @@
+# CAMARA OpenAPI Linting Rules Implementaion Guideline [ How to integrate the rules into CAMARA repository ]
+
+## Introduction
+
+This guide provides instructions on implement linting rules for the CAMARA API using two methods: <b>Spectral Linting</b> and <b>Megalinter with Spectral Linting.</B>
+
+CAMARA suggests the second method, incorporating Megalinter with Spectral.
+
+## Megalinter with Spectral Linting
+
+Megalinter is an open-source tool for CI/CD workflows that analyzes the consistency of code, IAC, configuration, and scripts in repository sources. Megalinter supports Spectral Linting.
+
+## Implementation Files
+
+<b><a href="https://github.com/camaraproject/Commonalities/blob/main/artifacts/linting_rules/lint_function/workflows/megalinter.yml">megalinter.yml </a></b>:- Contains the configuration of megalinter along with spectral.
+
+<b><a href="https://github.com/camaraproject/Commonalities/blob/main/artifacts/linting_rules/.spectral.yml"> .spectral.yml </a></b> :- Linting rules based on the<a href="https://github.com/camaraproject/Commonalities/blob/main/documentation/Linting-rules.md"> OpenAPI Specification</a>
+
+## GitHub Actions Integration
+
+1. Add <b>megalinter.yml</b> to GitHub action workflow --> .github/workflows
+
+   which include the configuration of megalinter and spectral for GitHub actions.
+
+2. Add<b> .spectral.yml</b> (Rules) File -> root location of repository
```

Comment:
2. Add<b> .spectral.yml</b> (rules) file to -> root location of repository

Reply:
I've updated the latest commit with the changes